locale/en/blog/release/v4.0.0.md

```diff
@@ -0,0 +1,152 @@
+
+Node.js v4.0.0 contains V8 v4.5, the same version of V8 shipping with the Chrome web browser today. This brings with it many bonuses for Node.js users, most notably a raft of new [ES6](https://nodejs.org/en/docs/es6/) features that are enabled by default including block scoping, classes, typed arrays, generators, Promises, Symbols, template strings, collections (Map, Set, etc.) and, new to V8 v4.5, arrow functions.
+
+Over the past year thanks to donations from companies such as Rackspace, DigitalOcean, Scaleway, miniNodes, Voxer, Joyent, CloudFlare, ARM Holdings, and individuals from the community, we've built an amazing test cluster that includes an impressive line-up of Linux variants, OS X, Windows, FreeBSD and SmartOS. New to Node.js, we also have first-class support for ARM processors. Our test suite must be continually passing on ARMv6, ARMv7 and the new 64-bit ARMv8 processors. So Node.js is officially ready for use by hobbyists and ARM server users alike.
+
+Node.js v4 also brings a plan for [long-term support (LTS)](https://github.com/nodejs/LTS/) and a regular release cycle. Release versioning now follows [SemVer](http://semver.org/), so expect increments of both minor and patch version over the coming weeks as we fix bugs and add features. What we won't be doing within the v4.x branch is adding breaking changes.
+
+However, we have mapped a quick initial path to our first LTS version which will come in October. At the same time, a new v5.x branch will be cut and we will begin making a new series of Stable releases available, starting with v5.0.0. These will likely contain a new version of V8 and any breaking changes that may be necessary. The v4.x line of releases will continue for an additional 18 months in LTS, with a focus on stability and security and no unnecessary additions in functionality. Beyond that, v4.x will move in to Maintenance for an additional 12 months where it will continue to receive attention for critical bugs and security fixes.
+
+In parallel, we will be releasing a new Stable line of releases every 6 months, one in October and one in April each year. Every second one of these, the April release, will continue life in LTS form by the time October comes around. This means that there will be overlapping LTS branches being maintained throughout the year, each receiving attention for a total of 30 months (LTS plus Maintenance). We believe that the Stable and LTS cycles provide the room for the project to continue its **innovation** and stay with the cutting-edge of JavaScript while also serving the need for **predictable long-term stability and security** demanded by the growing line-up of enterprise users proudly adopting Node.js as a key technology; some of who already count themselves as [members of the Node.js Foundation](https://nodejs.org/en/foundation/members/).
```

Comment:
Perhaps mention here or above that the stable line will have basically weekly-ish patch/minor releases?